data/db-schema/src/main/resources/net/corda/db/schema/messagebus/migration/db-message-bus-tables-v1.0.xml

```diff
@@ -27,6 +27,10 @@
             <column name="state" type="integer"/>
```

Comment:
Although it was not changed with this PR, it would be useful to add a comment about possible states and related values (and ideally that would be constrained)